### PR TITLE
WIP: moved tests to separated directory

### PR DESCRIPTION
**Pull request by roblaszczak**

### CODE REVIEW

File: pubsub/gochannel/doc.go [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// This is just simplest Pub/Sub implementation
```
Comment (m110) [inline pubsub/gochannel/doc.go]:
```suggestion
// This is just the simplest Pub/Sub implementation
```

File: pubsub/gochannel/doc.go [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+//
+// All Pub/Sub's implementations can be found in https://watermill.io/docs/pub-sub-implementations/
```
Comment (m110) [inline pubsub/gochannel/doc.go]:
```suggestion
// All Pub/Sub implementations can be found at https://watermill.io/docs/pub-sub-implementations/
```

File: pubsub/gochannel/pubsub_test.go [hunk 3]
```diff
@@ -8,12 +8,14 @@ import (
 	"testing"
 	"time"
 
+	"github.com/ThreeDotsLabs/watermill/pubsub/gochannel"
+
+	"github.com/ThreeDotsLabs/watermill/pubsub/tests"
```
Comment (m110) [inline pubsub/gochannel/pubsub_test.go]: This is just cosmetics, but could be sorted properly.

Comment (roblaszczak) [reply pubsub/gochannel/pubsub_test.go]: done****

**Pull request merged**